src/__tests__/third-party/styletron-react/pages/_document.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import Document, {
+  DocumentContext,
+  Head,
+  Html,
+  Main,
+  NextScript,
+} from 'next/document';
+import { Provider as StyletronProvider } from 'styletron-react';
+import { Server, Sheet } from 'styletron-engine-atomic';
+import type { RenderPageResult } from 'next/dist/next-server/lib/utils';
+import { styletron } from '../styletron';
+
+type Props = RenderPageResult & {
+  stylesheets: Sheet[];
+};
+
+export default class CustomDocument extends Document<Props> {
+  static async getInitialProps(ctx: DocumentContext) {
+    const page = ctx.renderPage((App) => (props) => (
+      <StyletronProvider value={styletron}>
```

Comment:
This code is never executed. We are missing this part from `NextJS` https://github.com/vercel/next.js/blob/canary/packages/next/next-server/server/render.tsx#L961 which doesn't seem too complicated, but is hard to add into our code.

Reply:
Are these errors triggered because of this `next-page-tester` missing implementation or the normally happen in a real app?